Core/Models/DynamicsBase.hpp

```diff
@@ -33,7 +33,6 @@ class DynamicsBase{
 
     using PositionBound = BoundBase<PositionDimensions, Scalar>;
     using PositionBoundPtr = std::shared_ptr<PositionBound>;
-
     const static unsigned int Dimension = Dimensions; 
```

Comment:
To undo the changes to this file, you can just cherry-pick it from `main`.

Reply:
Cherry-picked the function from main